Lib/fontbakery/profiles/googlefonts.py

```diff
@@ -4631,6 +4632,49 @@ def com_google_fonts_check_varfont_instance_names(ttFont):
     yield PASS, "Instance names are correct"
 
 
+@check(
+  id = 'com.google.fonts/check/varfont_duplicate_instance_names',
+  rationale = """
+    This check's purpose is to detect duplicate named instance names in a given variable font.
+
+    Repeating instance names may be the result of instances for several VF axes defined in `fvar`, but since 
+    currently only weight+italic tokens are allowed in instance names as per GF specs, they ended up repeating.
+
+    Instead, only a base set of fonts for the most default representation of the family can be defined through instances in the `fvar` table,
```

Comment:
what do you mean by "the most default representation"?

Reply:
For example, an optical size for text would be the default for an opsz axis. The word default was mentioned in the Hangout chats yesterday.